### PR TITLE
Improved jsdocs and documentation

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: docs/messages.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+title: Messages
+description: A superclass for the messages of the bitcoin network
+---
+# Messages
+
+The bitcoin protocol specifies a set of [messages](https://en.bitcoin.it/wiki/Protocol_specification) that can be sent from peer to peer. `bitcore-p2p` provides partial support for some of these messages.
```
Comment (maraoz) [inline docs/messages.md]: "provides partial support for some of these messages." sounds like only some messages are *incompletely* supported. I would change it to "provides support for some of these messages."

File: lib/messages.js [hunk 2]
```diff
@@ -171,20 +174,53 @@ module.exports.Message = Message;
 function Version(subversion, nonce) {
   this.command = 'version';
   this.version = PROTOCOL_VERSION;
-  this.subversion = subversion || '/BitcoinX:0.1/';
+  this.subversion = subversion || '/bitcore:0.8/';
```
Comment (maraoz) [inline lib/messages.js]:
```javascript
this.subversion = subversion || '/bitcore:'+require('package.json').version+'/';
```

**Comment by maraoz** [comment]
travis failed

**Comment by yemel** [comment]
ACK

**Pull request merged**